manifests/params.pp

```diff
@@ -0,0 +1,5 @@
+class foreman_scap_client::params {
+  $ca_file          = pick($::rh_certificate_consumer_ca_file, '/var/lib/puppet/ssl/certs/ca.pem')
+  $host_certificate = pick($::rh_certificate_consumer_host_cert, "/var/lib/puppet/ssl/certs/${fqdn}.pem")
+  $host_private_key = pick($::rh_certificate_consumer_host_key, "/var/lib/puppet/ssl/private_keys/${fqdn}.pem")
+}
```

Comment:
Yeah, had to add a newline, commit, squash, rebase and push - and github ignored my effort :bowtie: